spec/safer_rails_console/patches/sandbox/auto_rollback_spec.rb

```diff
@@ -0,0 +1,45 @@
+# require 'safer_rails_console/patches/sandbox/auto_rollback'
```

Comment:
Currently all the patches are integration tested in a real rails setup and console. We don't `require` anything within `patches` outside of that. See this context https://github.com/salsify/safer_rails_console/blob/master/spec/integration/patches/sandbox_spec.rb#L8 for more information. Feel free to help improve that by including the case where a normal exception is forwarded!

That being said, I'm totally for unit testing the patches. I'd probably update the test for `SaferRailsConsole::Console.initialize_sandbox` to the following:
```
  context ".initialize_sandbox" do
    before do
      allow(SaferRailsConsole::Console).to receive(:require)
    end

    it "loads sandbox patches" do
      described_class.initialize_sandbox

      expect(SaferRailsConsole::Console).to have_received(:require).with('safer_rails_console/patches/sandbox')
    end
  end
```